src/modules/print/__tests__/printComponent-test.js

```diff
@@ -79,8 +79,8 @@ describe('PrintTemplateComponent', () => {
     expect(renderTemplate.getCall(0).args[0].codeFormat).toBe(2);
     expect(renderTemplate.getCall(0).args[0].templateId).toBe(4);
     expect(renderTemplate.getCall(0).args[0].nodes).toEqual([
-      {'uuid':1,'name':'Utviklingsmuseet - Utviklingsmuseet Org - Forskningens hus - Forskningsværelset - kkk - Test 1'},
-      {'uuid':2,'name':'Utviklingsmuseet - Utviklingsmuseet Org - Test 2'}
+      {'uuid':1,'name':'Utviklingsmuseet / Utviklingsmuseet Org / Forskningens hus / Forskningsværelset / kkk / Test 1'},
+      {'uuid':2,'name':'Utviklingsmuseet / Utviklingsmuseet Org / Test 2'}
```

Comment:
legg til en test hvor path er tom, dette er tilfelle for root noder. uten fiksen skal man da få "- [node navn]" .. altså en bindestrek på starten uten mening. Dette var et problem på MUSARK-949 og det må lages en test for den defecten.